kalite/distributed/features/exercise.feature

```diff
@@ -3,11 +3,6 @@ Feature: Exercises on the Learn page
     Internally we distinguish between Khan exercises and perseus exercises,
     But the user doesn't and we shouldn't here.
 
-    Scenario: Exercise is not available
```

Comment:
Can you check the step implementation? I think this "feature" was intended as a proof of concept, so if it's not performing any useful checks then we can just remove this feature file entirely.